db/migrate/20170928152249_add_null_restraints_to_users.rb

```diff
@@ -1,8 +1,10 @@
 class AddNullRestraintsToUsers < ActiveRecord::Migration[5.1]
   def change
-    change_column_null :users, :password_digest, false
-    change_column_null :users, :username,        false
-    change_column_null :users, :email,           false
-    change_column_null :users, :full_name,       false
+    change_column_null :users, :password_digest,   false
+    change_column_null :users, :username,          false
+    change_column_null :users, :email,             false
+    change_column_null :users, :full_name,         false
+    change_column_null :users, :admin,             false
+    change_column_null :users, :membership_active, false
```

Comment:
Wasn't this migration already run a while back? I could be reading this incorrectly, but it looks like you changed the migration script. Can you talk me through this? Generally speaking, we shouldn't change an old migration. Instead, we should write a new migration that makes the changes we need.

Reply:
Ok, I'll be sure to do that in the future.